### DISCUSSION TITLE
v9.5.3 - Access-Control-Allow-Origin

### DISCUSSION THREAD

**Discussion by gentlemanZhang** [vercel/next.js#16819] [Help]
# When I use the NextJS of version v9.5.3.  In my pages on the production environment,  use "link" tag to preload the stylesheets. It will happend the CORS ERROR. I suggest  to lock the version of your package with some lock files.
![avatar](https://file.netspass.cn/image/20200903/6d15fe1cc6c1487eae80cf685de2433f.png)

**Comment by StephenHolmes14** [comment]
Having similar problems trying to implement a next js website as part of a micro front-end.

Might have to swap to something else because of it
